source/community.applications/usr/local/emhttp/plugins/community.applications/skins/Narrow/skin.html

```
@@ -1,113 +1,564 @@
+				<li><hr class='onlyShowWithFeed'/></li>
+			</ul>
+				<span id='category_list' class='onlyShowWithFeed'></span>
+			<ul class='caMenu'>
+				<li class='categoryMenu caMenuItem allApps onlyShowWithFeed nonDockerSearch' data-category='All'><span class='caAllAppsMenuLabel' data-all-apps-label="<?tr('All Apps');?>" data-all-results-label="<?tr('All Results');?>"><?tr("All Apps");?></span></li>
+				<li class='caMenuItem'><hr class='category_hr onlyShowWithFeed'></li>
+				<li class='categoryMenu caMenuItem caRepositoryMenu onlyShowWithFeed nonDockerSearch' data-category='repos'><?tr("Repositories");?></li>
+				<li><hr class='category_hr onlyShowWithFeed'/></li>
+				<li class='caMenuItem showSettings noSelect'><?tr("Settings");?></li>
+				<li class='caMenuItem showStatistics noSelect'><?tr("Statistics");?></li>
+				<li class='caMenuItem showCredits noSelect'><?tr("Credits");?></li>
+				<li class='caMenuItem'><a class='caMenuItem' href='https://forums.unraid.net/topic/38582-plug-in-community-applications/' target='_blank'><?tr("Support");?></a></li>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)